server/game/cards/12-KotI/WhatIsDeadMayNeverDie.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard.js');
+
+class WhatIsDeadMayNeverDie extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onDominanceDetermined: event => this.controller === event.winner
+            },
+            target: {
+                activePromptTitle: 'Select character',
+                cardCondition: card => this.cardCondition(card)
+            },
+            handler: context => {
+                context.player.putIntoPlay(context.target);
+                this.game.addMessage('{0} uses {1} to put {2} into play from their dead pile', context.player, this, context.target);
```

Comment:
This is a really small thing, but the general convention we've used is `{0} plays {1}` for event cards instead of `{0} uses {1}` which is used for everything else.